controllers/admin/AdminTranslationsController.php

```diff
@@ -85,9 +87,14 @@ public function __construct()
 
         $this->link_lang_pack = str_replace('%ps_version%', _PS_VERSION_, $this->link_lang_pack);
 
-        $this->themes = (new ThemeManagerBuilder($this->context, Db::getInstance()))
-            ->buildRepository()
-            ->getList();
+        $container = SymfonyContainer::getInstance();
+        if (null !== $container) {
+            $themeManagerBuilder = $container->get('prestashop.core.addon.theme.theme_manager_builder');
```

Comment:
Since we are in controllers, we can control what is loaded. The code is duplicated here.